plugins/auth-backend/src/providers/PassportStrategyHelper.ts

```diff
@@ -98,10 +134,22 @@ export const executeRefreshTokenStrategy = async (
             ),
           );
         }
-        resolve({
+
+        anyStrategy.userProfile(
```

Comment:
Talked about exposing this as a separate helper to let it be a bit more flexible in the provider implementation